src/components/Login/index.tsx

```diff
@@ -19,6 +25,7 @@ interface Props {
 
 function LoginModalContent(props: Props) {
   const initialRef = React.useRef(null);
+  const logoStyles = useStyleConfig("Logo");
 
   return (
     <Modal
```

Comment:
When I open the modal, the close button will auto focus
<img width="553" alt="image" src="https://user-images.githubusercontent.com/61776449/173229410-46e080e9-419c-4083-820c-ea0c972da52c.png">

Reply:
Try to find the way of using "initialFocusRef" in the Modal component on Chakra Doc can fix this.

Normally people would not like to focus on the close button after opening a modal because users may just hit the enter key and assume this is for the first option(Continue with Google in our case)

Reply:
You can refer to this part we did before.

Reply:
You can refer to this part we did before.cc-0035

Reply:
Fixed. Thank you all for your suggestion and advice.